modules/ROOT/pages/to-set-permissions.adoc

```diff
@@ -4,35 +4,37 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Exchange provides multiple ways to set up access management to meet governance requirements and at the same time drive collaboration across teams. Anypoint Platform comes with predefined role groups that can be set in the Access Management > Roles settings to provide users with different access levels.
+Exchange provides multiple ways to set up access management to provide collaboration across teams and meet governance requirements. Anypoint Platform comes with predefined permissions and role groups that can be set in Access Management to provide users with different access levels.
 
-== Exchange Role Groups
+== Exchange Permissions
 
-These role groups pertain to every Business Group level:
+Exchange permissions are the best way to control access to assets in a business group. Before Exchange permissions were introduced, very similar Exchange role groups controlled Exchange access, and role groups are still supported.
```

Comment:
Use Exchange permissions to control access to assets in a business group. You can assign Exchange permissions to individual users or by using the xref:access-management::teams.adoc[Teams] feature.

Reply:
I *think* you can use teams for this? Please verify.

Reply:
Fixed.